src/app/pages/BuyPage/components/SwapAssetSelector/Loadable.tsx

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
we don't need this file if we use one of global compoments

Reply:
fixed